File: sb_payment_sdk/src/main/java/com/startbutton/sb_payment_sdk/models/BankAccountModel.java
package com.startbutton.sb_payment_sdk.models;

public class BankAccountModel {
    private String id;
    private String accountNumber;
    private String accountName;
    private String bankName;

    public BankAccountModel(String id, String accountNumber, String accountName, String bankName){
        this.id = id;
        this.accountNumber = accountNumber;
        this.accountName = accountName;
        this.bankName = bankName;
    }

    public String getId () {
        return this.id;
    }

    public void setId (String id) {
        this.id = id;
    }

    public String getAccountNumber() {
        return accountNumber;
    }

    public void setAccountNumber(String accountNumber) {
        this.accountNumber = accountNumber;
    }

    public String getAccountName() {
        return accountName;
    }

    public void setAccountName(String accountName) {
        this.accountName = accountName;
    }

    public String getBankName() {
        return bankName;
    }

    public void setBankName(String bankName) {
        this.bankName = bankName;
    }
}

File: lib/models/bank_account_model.dart
class BankAccountModel {
  final id;
  final accountNumber;
  final accountName;
  final bankName;

  BankAccountModel({this.id, this.accountNumber, this.accountName, this.bankName});

  factory BankAccountModel.fromJson(Map<String, dynamic> data){
    return BankAccountModel(
      id: data["id"],
      accountName: data["accountName"],
      accountNumber: data["accountNumber"],
      bankName: data["bankName"],
    );
  }
}